### PR TITLE
Adding highest/lowest lists to MapCard

### PR DESCRIPTION
**Pull request by kkatzen**
Addresses #826

There is some tacky blinking when the vega map has to reload...  Not really sure how much can be done about it but we can look into it
Screen Shot 2021-05-12 at 1 09 18 PM

### CODE REVIEW

**Comment by joshzarrabi** [comment]
This is great! Maybe we should get rid of the map reload until we figure out how to make it fast? @vanshkumar what do you think?

File: frontend/src/cards/MapCard.tsx [hunk 1]
```diff
@@ -125,6 +130,17 @@ function MapCardWithKey(props: MapCardProps) {
             (row: Row) => row[activeBreakdownVar] === activeBreakdownFilter
           );
 
+        const highestFive = dataForActiveBreakdownFilter
+          .sort((rowA: Row, rowB: Row) =>
+            rowA[metricConfig.metricId] <= rowB[metricConfig.metricId] ? 1 : -1
+          )
+          .slice(0, 5);
```
Comment (vanshkumar) [inline frontend/src/cards/MapCard.tsx:137]: Can we make the 5 into a constant at the top, and then also change the variable names here to mostAffectedGeos or something like that?

**Review by vanshkumar — COMMENTED** [review]
> This is great! Maybe we should get rid of the map reload until we figure out how to make it fast? @vanshkumar what do you think?

Yeah, that's where I lean as well, though let's get feedback from SHLI folks in the sync w Daniel tomorrow morning

Thanks Krista, this is awesome! Just a note, I imagine we will want to do this for the unknowns map card later as well & at that time it makes sense to refactor this into something reusable, but this looks great for now!

File: frontend/src/cards/MapCard.tsx (hunk 1 above)
Comment (kkatzen) [reply frontend/src/cards/MapCard.tsx:137]: Why the variable name change? What about highestRates and lowestRates ? I think mostAffectedGeos is less clear if you think about other metrics like "uninsured individuals" i think highest rates of uninsured is more straightforward than most affected geos for uninsured ?

Comment (vanshkumar) [reply frontend/src/cards/MapCard.tsx:137]: Suggesting the variable name change as the 5 will become a constant which can then be updated independently

Good point, highestRates/lowestRates sgtm

Comment (kkatzen) [reply frontend/src/cards/MapCard.tsx:137]: ah gotcha word

Comment (vanshkumar) [reply frontend/src/cards/MapCard.tsx:137]: Oh, I see now that the text "Top 5 Highest Rates" has a hard-coded 5 as well, we should use the constant there as well if possible - if it's not possible, I'm fine with leaving everything as-is (ie leave the 5 hard coded everywhere)

**Review by vanshkumar — APPROVED** [review]

**Comment by kkatzen** [comment]
I moved the list into its own component so it'll be faster to add it to unknowns if we want to.

I left the map as it is - also worth noting the tooltips say "no data" for all places when it is expanded and we would probably want to change that, right? I was thinking I'd just leave the map updating for now so you can show Daniel the PR tomorrow and then we can submit without the map changing and follow up with a neater version?

**Review by joshzarrabi — APPROVED** [review]

**Pull request merged**